bears/vcs/git/GitCommitBear.py

```diff
@@ -66,6 +98,7 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             self.err('git:', repr(stderr))
             return
 
+        body = stdout.rstrip('\n')
```

Comment:
introduction of a `shortlog` and `body` variable could be also done in a separate commit :+1: